app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -122,6 +132,16 @@ def mfa_verify_phone_otp_submitted(reauthentication:, success:)
       )
     end
 
+    # @param [String] the type of multi-factor authentication used
+    # The user has exceeded the rate limit during verification
+    # and account has been locked
+    def mfa_verify_rate_limited(type)
+      track_event(
+        :mfa_verify_rate_limited,
+        type: type,
+      )
+    end
```

Comment:
Oh, hmm -- I know I didn't bring this up earlier, but should be just call this `mfa_login_rate_limited` now from the start? I think that's the approach @olatifflexion took in his PR.

Reply:
done